src/gsi/hybrid_ensemble_isotropic.F90

```diff
@@ -3825,54 +3903,93 @@ subroutine ckgcov_a_en_new_factorization(ig,z,a_en)
      call stop2(999)
   endif
 
+! MGBF-based localization
+  if(l_mgbf_loc) then
```

Comment:
@shoyokota  I suggest to add a comment, that it is mandated that l_mgbf_loc =.true., only when regional=.false.

Reply:
Now I added the comment.

Reply:
In l_mgbf_loc branch, what will happen when "if(grd_loc%kend_loc+1-grd_loc%kbegin_loc==0)  is true?

Reply:
This conditional branch is required only in case vertically-parallelized domain may be empty in some processors. This condition is required in vertically-parallelized RF. However, it is not required in horizontally-parallelized MGBF. Namely, we can apply MGBF regardless of grd_loc%kend_loc+1-grd_loc%kbegin_loc==0 or not.